### ISSUE TITLE
Doesn't work :shrug:

### ISSUE THREAD

**Issue by gromakovsky** [serokell/yt-utilities#22]
Org file:
```
* Cardano-SL
** IN-PROGRESS CSL-1203 Test basic delegation with tx distribution
*** Review PR #1329
    CLOCK: [2017-08-14 Mon 13:11]--[2017-08-14 Mon 13:29] =>  0:18
```
Output:
> WARNING: Found a valid CLOCK inside yt subtree that is not related to YT taskClockRecord "" 2017-08-14 13:11:00 UTC 2017-08-14 13:29:00 UTC

So for some reason it's not parsed correctly 😒.

**Comment by gromakovsky** [comment]
See #23.

**Comment by gromakovsky** [comment]
Fixed in #23.
